stripe/index.js

```diff
@@ -23,8 +23,12 @@ class Stripe {
 
   async getStripeCustomerDonationInfo ({ customerId }) {
     const customer = await this.getStripeCustomer({ customerId })
-    const donationInfo = {
-      last4: customer.sources.data[0].last4
+    const donationInfo = {}
+    // This will reject if the customer has no billing sources, hence the try catch
+    try {
+      donationInfo.last4 = customer.sources.data[0].last4
+    } catch (e) {
+      donationInfo.last4 = 'n/a'
```

Comment:
great use case for lodash.get :)

Reply:
MMWT

Reply:
i did optional chaining but it broke locally due to node versions and i wasn't confident of our node version in the cloud so opted for try catch